Core/clim-core/table-formatting.lisp

```diff
@@ -259,66 +325,35 @@ skips intervening non-table output record structures."))
-  (gen-invoke-trampoline 'invoke-formatting-table
-                         (list (stream-designator-symbol stream
-                                                         '*standard-output*))
-                         args
-                         body))
-
-(defun invoke-formatting-table
-    (stream continuation
-     &key x-spacing y-spacing
-       multiple-columns
-       multiple-columns-x-spacing
-       equalize-column-widths
-       (move-cursor t)
-       (record-type 'standard-table-output-record)
-       &allow-other-keys)
-  (setq x-spacing (parse-space stream (or x-spacing #\Space) :horizontal))
-  (setq y-spacing (parse-space stream (or y-spacing
-                                          (stream-vertical-spacing stream))
-                               :vertical))
-  (setq multiple-columns-x-spacing
-        (if multiple-columns-x-spacing
-            (parse-space stream multiple-columns-x-spacing :horizontal)
-            x-spacing))
-  (with-new-output-record
-      (stream record-type table
-              :x-spacing x-spacing
-              :y-spacing y-spacing
-              :multiple-columns multiple-columns
-              :multiple-columns-x-spacing multiple-columns-x-spacing
-              :equalize-column-widths equalize-column-widths)
-    (multiple-value-bind (cursor-old-x cursor-old-y)
-        (stream-cursor-position stream)
-      (with-output-recording-options (stream :record t :draw nil)
-        (funcall continuation stream)
-        (force-output stream))
-      (setf (stream-cursor-position stream)
-            (values cursor-old-x cursor-old-y))
-      (with-output-recording-options (stream :record nil :draw nil)
-        (adjust-table-cells table stream)
-        (when multiple-columns (adjust-multiple-columns table stream))
-        (tree-recompute-extent table))
-      (replay table stream)
-      (if move-cursor
-          (setf (stream-cursor-position stream)
-                (values (bounding-rectangle-max-x table)
-                        (bounding-rectangle-max-y table)))
-          (setf (stream-cursor-position stream)
-                (values cursor-old-x cursor-old-y))))))
+(define-formatting-macro (formatting-table standard-table-output-record)
+    ((move-cursor                :default   t :passp nil)
+     (x-spacing                  :transform (parse-space stream (or x-spacing #\Space) :horizontal))
+     (y-spacing                  :transform (parse-space stream (or y-spacing
+                                                                    (stream-vertical-spacing stream))
+                                                         :vertical))
+     (multiple-columns           :bindp t)
+     (multiple-columns-x-spacing :transform (if multiple-columns-x-spacing
+                                                (parse-space stream multiple-columns-x-spacing :horizontal)
+                                                x-spacing))
+     (equalize-column-widths))
```

Comment:
don't align forms in columns (applies to other definitions too). If you want to have them aligned, start each argument from a new line, like
```
(move-cursor
 :default t
 :passp nil)
```

or simply remove extra spaces.

Reply:
Changed.